Plugins/RSBot.Map/Views/Main.cs

```diff
@@ -655,6 +655,9 @@ private void mapCanvas_MouseClick(object sender, MouseEventArgs e)
         private void checkEnableCollisions_CheckedChanged(object sender, EventArgs e)
         {
             GlobalConfig.Set("RSBot.EnableCollisionDetection", checkEnableCollisions.Checked);
+
+            if (Game.Player != null)
+                CollisionManager.Update(Game.Player.Position.Region.Id);
```

Comment:
Please remove the added lines. Because its checking in Kernel.cs dont need extra checking at here.

Reply:
The kernel will only update the region if the Id is different. So if you stay in the same region and activate/deactivate collision checking, the region will not be refreshed at all. That's why I'm assuring to update the collisions here.

Reply:
Removed it after changes to the kernel.cs